### ISSUE TITLE
How i can get the text of the response of the bot?

### ISSUE THREAD

**Issue by AlumnoZ** [ahmadfaizalbh/Chatbot#55]
So im trying to transfer this bot to window with text, but i really dont know how?

**Comment by ahmadfaizalbh** [comment]
If you are building application for deployment then make sure to use DB to override python data structure so that you can serve through http/https and all state, topic, conversations and memory are in DB associated with user you can checkout [ChatBotAI-HTTPServer](https://github.com/ahmadfaizalbh/ChatBotAI-HTTPServer) or [WebBot](https://github.com/ahmadfaizalbh/WebBot) using [Django wrapper](https://github.com/ahmadfaizalbh/Django-Chatbot)

Basically `respond` method takes user message (test) and session_id (text: user name) and returns a text

```python
bot_response = chat.respond(user_message, session_id)
```

**Comment by ahmadfaizalbh** [comment]
if you are planning to use it with Microsoft bot framework then checkout [Microsoft-chatbot](https://github.com/ahmadfaizalbh/Microsoft-chatbot)

**Comment by AlumnoZ** [comment]
Thanks
